pkg/webui/console/store/index.js

```diff
@@ -12,20 +12,88 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { mapValues } from 'lodash'
+import * as Sentry from '@sentry/browser'
 import { createStore, applyMiddleware, compose } from 'redux'
 import { createLogicMiddleware } from 'redux-logic'
 import { routerMiddleware } from 'connected-react-router'
+import createSentryMiddleware from 'redux-sentry-middleware'
 
 import dev from '@ttn-lw/lib/dev'
+import env from '@ttn-lw/lib/env'
 
 import createRootReducer from './reducers'
 import requestPromiseMiddleware from './middleware/request-promise-middleware'
 import logics from './middleware/logics'
 
+if (env.sentryDsn)
```

Comment:
That shouldn't be necessary, no? Sentry was already initialized in `console.js`. There should be away to get the already instantiated Sentry object.

Reply:
Also let's not use non-bracketed if statements when followed by a mult-line statement. Generally, I think we should use non-bracketed if statements only sparsely. I'm ok with using then if they are e.g. followed by a simple return statement.